README.md

```diff
@@ -1,7 +1,11 @@
 # GoogleUdacity_JeevanRakht
 
-The code and development of the JeevanRakht's website is housed here.   
-For more details - [Refer to this paper.](https://paper.dropbox.com/doc/JeevanRakht-WebD-Project-ZroUbusvKbwRSRCDFHCOV)
+JeevanRakht is a Virtual Blood Bank web app, that aims at providing mechanism for people in need to directly talk to willing donors, and also find out blood availability in nearby blood banks.
+On this platform people willing to donate blood can sign up with details of their blood groups, location. Also there’ll be accounts of blood banks with details like number of packages of blood available and blood group, location of blood bank to filter out nearest blood banks.
+You can find the project's wireframe below which depicts the basic flow of our web application.
+
+## Classified Documents (Access restricted to Mods/Co-Mods only)
```

Comment:
I think,  you can remove the private section altogether and you will be good to go and merge.

Reply:
okay will remove it and merge.